sdk/@launchdarkly/observability-dotnet/src/LaunchDarkly.Observability/LaunchDarkly.Observability.csproj

```diff
@@ -35,4 +34,31 @@
     <AssemblyOriginatorKeyFile>../../../../../LaunchDarkly.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
   </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Open Telemetry Dependencies -->
+    <PackageReference Include="OpenTelemetry" Version="1.12.0" />
+    <PackageReference Include="OpenTelemetry.Exporter.Console" Version="1.12.0" />
+    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.12.0" />
+    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.12.0" />
+    <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.12.0" />

```

Comment:
I've directly added each dependency the SDK is using. In highlight these were transient dependencies of the auto-instrumentation.

The auto-instrumentation doesn't appear to have a programmatic way to instrument things, so I figured it would be better to limit the default instrumentations.

That said I think we could extend the builder to easily allow adding more instrumentations to the tracer.

```
ldconfig.WithTraceInstrumentation(tracing => {
  tracing.AddWhateverInstrumentation();
})
```

Reply:
makes sense to me